src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+
+const SearchBar: React.FC = () => {
+  return (
+    <form action="">
+      <div className="p-1 bg-light rounded rounded-pill shadow-sm mb-4">
+        <div className="input-group">
+          <input
+            type="search"
+            placeholder="What're you searching for?"
```

Comment:
useText()

Reply:
Made all the necessary changes sorry for the mistake